pkg/crc/systemd/systemd.go

```diff
@@ -65,6 +65,14 @@ func (c Commander) DaemonReload() error {
 	return nil
 }
 
+func (c Commander) EnableAndStart(name string) error {
+	if _, err := c.service(name, actions.Enable); err != nil {
+		return err
+	}
+	_, err := c.service(name, actions.Start)
+	return err
+}
```

Comment:
Why? Chaining 2 methods from the public API in a method which will be used once does not feel like something which deserves its own helper

Reply:
@cfergeau I wanted to use it for podman systemd unit and libvirtd unit also but looks like in case of libvirtd we don't enable the socket/service. For podman side we we just need to enable it once and run if not running, will drop this one.